{{cookiecutter.project_slug}}/README.md

```diff
@@ -17,12 +19,26 @@ To activate the environment run:
 To update this environment with your production dependencies run:
 
     conda env update --file=environment.yml
-{% else %}
+{% elif cookiecutter.package_manager == 'pip' %}
+To set up your local development environment, please use a fresh virtual environment.
+
 Then run:
 
     pip install -r requirements.txt -r requirements-dev.txt
-{% endif %}
-You can now {% if cookiecutter.create_cli == 'yes' %}run the module from the `src` directory with `python -m {{ cookiecutter.module_name }}`{% else %}import functions and classes from the module with `import {{ cookiecutter.module_name }}`{% endif %}.
+{% else %}
+To set up your local development environment, run:
+
+    poetry install
+
+Behind the scenes, this creates a virtual environment and installs your dependencies into it. Whenever you run `poetry run <command>`, that `<command>` is actually run inside the virtualenv managed by poetry.
```

Comment:
@klamann I added this as an explanation of `poetry run`. Do you think that's sufficient, or should I add further explanation closer to `poetry run jupyter notebook`?